tensorflow_datasets/core/features/bounding_boxes.py

```diff
@@ -79,10 +82,43 @@ def encode_example(self, bbox):
         [bbox.ymin, bbox.xmin, bbox.ymax, bbox.xmax]
     )
 
+  def repr_html(self, ex: np.ndarray) -> str:
+    """Returns the HTML str representation of an Image with BBoxes."""
+    ex = np.expand_dims(ex, axis=0)  # Convert single bounding box to single batch.
+    return self._repr_html(ex)
+
+  def repr_html_batch(self, ex: np.ndarray) -> str:
+    """Returns the HTML str representation of an Image with BBoxes (Sequence)."""
+    return self._repr_html(ex)
+
+  def _repr_html(self, ex: np.ndarray) -> str:
```

Comment:
If `self` isn't used anywhere, this should be a simple function. This helps readability by explicitly showing there is no hidden magic/`self` input.